### PR TITLE
Add InterfaceOperStatus deviation

### PR DESCRIPTION
**Pull request by mojiiba**
This pull adds a deviation InterfaceOperStatus that is related to devices that generate `Interface_OperStatus_DOWN` instead of `Interface_OperStatus_LOWER_LAYER_DOWN` for an aggregated link.

### CODE REVIEW

**Review by bstoll — APPROVED** [review]
Thanks, this change SGTM.

File: internal/deviations/deviations.go [hunk 1]
```diff
@@ -69,6 +69,9 @@ import "flag"
 var (
 	InterfaceEnabled = flag.Bool("deviation_interface_enabled", false,
 		"Device requires interface enabled leaf booleans to be explicitly set to true.  Full OpenConfig compliant devices should pass both with and without this deviation.")
+
```
Comment (bstoll) [inline internal/deviations/deviations.go:72]: gofmt (CI checks) don't like the stray characters on this line

Comment (mojiiba) [reply internal/deviations/deviations.go]: thanks!, fixed the issue.

**Review by bstoll — DISMISSED** [review]

**Review by amrindrr — APPROVED** [review]

**Pull request merged**